src/frontend/js/types/Enums.ts

```diff
@@ -11,3 +11,9 @@ export enum filterApiRole {
   NOT_MODERATOR = '!moderator',
   MODERATOR = 'moderator',
 }
+
+export enum FileUploadError {
+  NONE = 0,
+  MAX_UPLOAD = 1,
+  ERROR = 2,
+}
```

Comment:
You could use string. Use integer like you did is implicit (https://www.typescriptlang.org/docs/handbook/enums.html#numeric-enums).

In the case of errors, it could useful to use string instead of integer (e.g: if you send to improve readability of errors in Sentry)

```suggestion
export enum FileUploadError {
  NONE = 'NONE',
  MAX_UPLOAD = 'MAX_UPLOAD',
  ERROR = 'ERROR',
}
```